pkg/app/app.go

```diff
@@ -530,6 +530,23 @@ func (a *App) Test(c TestConfigProvider) error {
 	}, false, SetFilter(true))
 }
 
+func (a *App) PrintDAGState(c StateConfigProvider) error {
+	var err error
+	return a.ForEachState(func(run *Run) (ok bool, errs []error) {
+		err = run.withPreparedCharts("show-dag", state.ChartPrepareOptions{
+			SkipRepos:   true,
+			SkipDeps:    true,
+			Concurrency: 2,
```

Comment:
should we need flags for those options?

Reply:
Added flags for skipDeps and concurrency